changelogs/8.15.asciidoc

```diff
@@ -28,6 +31,10 @@ https://github.com/elastic/apm-server/compare/v8.15.0\...v8.15.1[View commits]
 [float]
 ==== Added
 
+- Self-instrumentating distributed traces can be emitted in OpenTelemetry format, if configured to do so
```

Comment:
This is about https://github.com/elastic/go-docappender/pull/182 right? It is not relevant to apm-server.
```suggestion
```